tools/dockerfile/interoptest/grpc_interop_cxx/build_interop.sh

```diff
@@ -31,7 +31,7 @@ cd /var/local/git/grpc
 make install-certs
 
 # build C++ interop client & server
-make interop_client interop_server
+make interop_client interop_server -j8
```

Comment:
not sure about this:
Currently 8 interop images are going to be built in parallel and 8*8 = 64 (the workers have 16 cores).
https://github.com/grpc/grpc/blob/912b8ab4d49cd6cde76675d37c896e5edcf67487/tools/internal_ci/linux/grpc_interop_tocloud.cfg#L29

Change to `-j2` which gives some boost and is conservative enough?